tests/test_data/projects/snowpark_functions/snowflake.yml

```diff
@@ -0,0 +1,14 @@
+definition_version: 1
+functions:
+  - name: func1
+    handler: "app.func1_handler"
+    signature:
+      - name: "a"
+        type: "string"
+      - name: "b"
+        type: "variant"
+    returns: string
+  - name: func2
+    handler: "app.func2_handler"
+    signature: ""
```

Comment:
`"signature": Seq(Argument) | EmptyList(),` 🤔

Reply:
`""` is representation of empty list according to strictyml, if I use `[]` it screams about "json like thing"

Reply:
🤯